matplotcheck/base.py

```diff
@@ -15,6 +15,7 @@
 from scipy import stats
 import pandas as pd
 import geopandas as gpd
+import datetime
```

Comment:
interesting. it datetime called explicitly in this module @ryla5068 ? i know when it inherits it will be required but that is in timeseries. i'm just curious how it was working previously!

Reply:
This was left in from when I was trying to convert times in the base module. I just pushed a commit removing it.

Reply:
great!! thanks @ryla5068 !!!